server/data/referenceDataClient.ts

```diff
@@ -12,4 +13,8 @@ export default class ReferenceDataClient {
   async getReferenceData(objectType: string): Promise<Array<ReferenceData>> {
     return (await this.restClient.get({ path: `/reference-data/${objectType}` })) as Array<ReferenceData>
   }
+
```

Comment:
is there any specific reason for adding getProbationRegions since for rest of reference data we are directly calling getReferenceData

Reply:
there is a note in the commit message: 
"Probation Regions have a different type signature to the rest of the Reference data.
Rather than having to handle this in the consuming code it makes sense that they have their
own method". 
If we use the same endpoints for both types of data we have to check which type of data it is in all places we call `getReferenceData`